tutorial/README.rst

```diff
@@ -13,5 +13,13 @@ The order we assume is as follows:
 * :ref:`pruning`
 * :ref:`sampler`
 
+Other Resources
```

Comment:
Just curious but are you expecting other items besides "examples" under this header?

Reply:
imho, the conference talk videos are candidates.